code/modules/mob/living/carbon/racial_spells.dm

```diff
@@ -0,0 +1,35 @@
+//For spells designed to be inherent abilities given to mobs via their species datum
+
+/spell/racial
```

Comment:
What's the point of this type? Racial spells have no reason to be grouped together, and you're restricting them from being aoe_turf or targeted.